CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to this project are documented in this file.
 - Add the support of QP problems with no constraint in `QPInverseKinematics` and `QPTSID` (https://github.com/ami-iit/bipedal-locomotion-framework/pull/784)
 - Implement `blf-joints-grid-position-tracking` application in `utilities` (https://github.com/ami-iit/bipedal-locomotion-framework/pull/787)
 - Add the possibility to resample the contact in a given contact list (https://github.com/ami-iit/bipedal-locomotion-framework/pull/788)
+- Set submodel states from IMUs in RDE (Use IMU measurements to set submodel base state in RDE)
```

Comment:
Can you rebase the PR on top of master and move this line to unreleased?

Reply:
Done